jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -38,11 +37,11 @@
  * <pre>{@code
  * Jib.from(baseImage)
  *    .addFiles(sourceFiles, extractionPath)
- *    .setEntrypoint(entrypoint)
- *    .setProgramArguments(arguments)
- *    .addEnvironmentVariable(key, value)
- *    .addExposedPort(
- *    .addLabel(key, value)
+ *    .setEntrypoint("myprogram", "--flag", "subcommand")
+ *    .setProgramArguments("hello", "world")
+ *    .addEnvironmentVariable("HOME", "/app")
+ *    .addExposedPort(8080)
```

Comment:
You may need a different example here since you changed it to use the `Port` class.

Reply:
Ah right